contrib/files-touched-check

```diff
@@ -7,6 +7,7 @@ while read -r status name; do
 
     case "$name" in
         README.md) continue ;;
+        .cirrus.yml) continue ;;
         contrib/*) continue ;;
         21.99-guixtest1/*) continue ;;  # guixtest1 used a different hierarchy
```

Comment:
Do you want to also remove the `21.99-guixtest1/` line, now that it [has been removed](https://github.com/bitcoin-core/guix.sigs/commit/bf94765a3472fe32337a8797744677b386fccd61) from this repo.

Reply:
Done!